R/derive_vars_aage.R

```diff
@@ -232,9 +232,8 @@ NULL
 
 #' @rdname derive_agegr_fda
 #' @export
-#' @details `derive_agegr_fda` Derive age groups according to FDA
-#' (\url{https://prsinfo.clinicaltrials.gov/results_definitions.html} ->
-#' Baseline Measure Information).
+#' @details `derive_agegr_fda` Derive age groups according to FDA. `age_var` will
+#'  be split in categories: <18, 18-64, >=65.
```

Comment:
Same comment as above, please insert a set of parentheses after the function name.